src/mpyl/cli/build.py

```diff
@@ -84,8 +85,15 @@ async def warn_if_update(console: Console):
     show_default=True,
     help="Verbose output",
 )
+@click.option(
+    "--cached",
+    help="Use cached build artifacts",
+    is_flag=True,
+    required=False,
+    default=False,
+)
```

Comment:
I think I would find it cleaner to leave the caching outside the build step. It's optional anyway, so thing will still work if you forget to d it.

Reply:
Actually @koeves proposed to put it in build since it can be used by all build commands